c2corg_ui/templates/route/edit.html

```diff
@@ -629,13 +629,19 @@ <h2 class="heading show-phone" translate>Comments</h2>
             </div>
 
             ## GLACIER GEAR
-          <div class="form-group data half" ng-class="{ 'has-success': route.glacier_gear }"
-               ng-if="route.activities.indexOf('skitouring') > -1 || route.activities.indexOf('snow_ice_mixed') > -1 || route.activities.indexOf('mountain_climbing') > -1 || route.activities.indexOf('rock_climbing') > -1
+            <div class="form-group data half" ng-class="{ 'has-success': route.glacier_gear }"
+                 ng-if="route.activities.indexOf('skitouring') > -1 || route.activities.indexOf('snow_ice_mixed') > -1 || route.activities.indexOf('mountain_climbing') > -1 || route.activities.indexOf('rock_climbing') > -1
                 || route.activities.indexOf('ice_climbing') > -1  || route.activities.indexOf('snowshoeing') > -1">
-            <label translate>glacier_gear</label>
-            <select class="form-control" ng-model="route.glacier_gear" ng-options="rat as mainCtrl.translate(rat) for rat in ${glacier_gear_types}"><option value=""></option></select>
-            <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="route.glacier_gear"></span>
-          </div>
+              <label translate>glacier_gear</label>
+              <select class="form-control" ng-model="route.glacier_gear" ng-options="rat as mainCtrl.translate(rat) for rat in ${glacier_gear_types}"><option value=""></option></select>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="route.glacier_gear"></span>
+            </div>
+
+            ## EXTERNAL RESOURCES
+            <div class="form-group data full">
+              <label translate>external_resources</label>
+              <textarea name="external_resources" ng-model="route.locales[0].external_resources" class="form-control" placeholder="{{'Books and websites not already associated to this route' | translate}}"></textarea>
```

Comment:
I find my initial string a bit clearer.
For instance we don't want that people fill this field in with the title of a book already associated to the route.